cdap-client/src/main/java/co/cask/cdap/client/ProgramClient.java

```diff
@@ -389,6 +391,27 @@ public void setProcedureInstances(String appId, String procedureId, int instance
   }
 
   /**
+   * Gets the number of instances of a service handler.
```

Comment:
I think now that they're separated we should just refer to them as `Service` and not "service handler". This is a general comment for all javadocs/documentation.

The reason is that you can't set instances for individual handlers, but only for the service as a whole. Also, now that workers have been moved out we don't need to distinguish the types.